src/GovUk.Education.ExploreEducationStatistics.Public.Data.Utils.Tests/TimePeriodFormatterTests.cs

```diff
@@ -77,17 +85,12 @@ public void InvalidSecondYear_Throws(string period)
             Assert.Throws<ArgumentOutOfRangeException>(() => TimePeriodFormatter.FormatFromCsv(period));
         }
 
-        [Fact]
-        public void SingleYear_Success()
-        {
-            Assert.Equal("2024/2025", TimePeriodFormatter.FormatFromCsv("202425"));
-        }
-
         [Theory]
         [InlineData("199899", "1998/1999")]
         [InlineData("199900", "1999/2000")]
         [InlineData("200001", "2000/2001")]
         [InlineData("200102", "2001/2002")]
+        [InlineData("202425", "2024/2025")]
```

Comment:
not sure we've gained anything by moving this test here. My previous comment was referring to testing a successful scenario where we just supply 1 year like `2020`, instead of a 6 digit year.

Or will we never have a 4 digit year?

Reply:
Ah good point, yep I'll add that in!